solvers/cyanure.py

```diff
@@ -4,6 +4,7 @@
 
 with safe_import_context() as import_ctx:
     from cyanure import Regression
+    import scipy
```

Comment:
You need to add `'scipy'` as a requirement in `requirements` of the class.

Reply:
good catch